policy/pipeline/artifacthub-pkg.yml

```diff
@@ -0,0 +1,19 @@
+# from https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml
+version: 0.0.1
+name: OPA Pipeline Policies
+displayName: OPA pipeline Policies
+createdAt: 2023-01-05T21:27:55.549418+00:00
+description: OPA policies enforcing standards on a tekton pipeline definition
+readme: |
+  OPA policies enforcing standards on a tekton pipeline definition
```

Comment:
Can we add a sentence and mention stone soup, with maybe a disclaimer to say it's perhaps not prod ready.

Reply:
It could also include the url to our docs page and/or or our github org.

Reply:
I can add something about stone soup. On line #17 the homeUrl is defined. Is that enough?

Reply:
Actually, is it a good idea to mention stone soup? I mean it could change pretty quick and the name isn't really anywhere on github

Reply:
I think the boat has sailed, i.e. look at https://github.com/hacbs-contract/ for example.

Reply:
But yeah, I think the url is good enough.